ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -287,6 +288,7 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
           dispatch(toggleNetworkMenu());
           dispatch(updateCustomNonce(''));
           dispatch(setNextNonce(''));
+          dispatch(detectNfts());
 
```

Comment:
We are caching the calls to the NFT-API so its okay if this call is made multiple times